.eslintrc.yml

```diff
@@ -7,6 +7,7 @@ extends:
   - plugin:import/typescript
   - plugin:@typescript-eslint/eslint-recommended
   - plugin:@typescript-eslint/recommended
+  - prettier
```

Comment:
`extending to prettier is needed so that prettier overrides conflicting rules automatically.